tests/functional/amo_sign_up.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'tests/functional/lib/helpers',
+  'tests/functional/lib/fx-desktop'
+], function (intern, registerSuite, FunctionalHelpers, FxDesktopHelpers) {
+
+  var MIGRATE_PARAMS = 'state=state&migration=amo&scope=scope';
+  var listenForFxaCommands = FxDesktopHelpers.listenForFxaCommands;
+
+  registerSuite({
+    name: 'Firefox AMO sign_up',
+
+    beforeEach: function () {
+      return FunctionalHelpers.clearBrowserState(this);
+    },
+
+    'as a migrating user': function () {
+      return FunctionalHelpers.openFxaFromRp(this, 'signup', MIGRATE_PARAMS)
+        .setFindTimeout(intern.config.pageLoadTimeout)
```

Comment:
`setFindTimeout` is taken care of by `openPage` via `openFxaFromRp`, no need for it here.